src/server/dflycmd.cc

```diff
@@ -84,17 +84,17 @@ OpStatus WaitReplicaFlowToCatchup(absl::Time end_time, shared_ptr<DflyCmd::Repli
       LOG(WARNING) << "Couldn't synchronize with replica for takeover in time: " << replica->address
                    << ":" << replica->listening_port << ", last acked: " << flow->last_acked_lsn
                    << ", expecting " << shard->journal()->GetLsn();
-      return OpStatus::TIMED_OUT;
+      return false;
```

Comment:
Why do you decide to remove OpStatus

Reply:
We don't care about the result in the only place which calls it, so why bother?

Reply:
Maybe we need to add this logic? these are different reasons for the fail

Reply:
Add logic for handling the different failure reasons? If so:
1. Why? Is there a bug today? (I imagine it's not because of this PR)
2. This is beyond the scope of this PR, which attempts to fix specific bugs